code/game/objects/effects/effect_system/smoke.dm

```diff
@@ -128,22 +132,73 @@
 
 /obj/effect/particle_effect/smoke/bad/Move()
 	. = ..()
-	for(var/mob/living/carbon/M in get_turf(src))
-		affect(M)
+	for(var/mob/living/carbon/affected_mob in get_turf(src))
+		affect(affected_mob)
 
-/obj/effect/particle_effect/smoke/bad/affect(mob/living/carbon/M)
+/obj/effect/particle_effect/smoke/bad/affect(mob/living/carbon/affected_mob)
 	..()
-	if (M.internal != null && M.wear_mask && (M.wear_mask.flags_inventory & ALLOWINTERNALS))
+	if (affected_mob.internal != null && affected_mob.wear_mask && (affected_mob.wear_mask.flags_inventory & ALLOWINTERNALS))
 		return
 	else
 		if(prob(20))
-			M.drop_held_item()
-		M.apply_damage(1, OXY)
-		if(M.coughedtime != 1)
-			M.coughedtime = 1
-			if(ishuman(M)) //Humans only to avoid issues
-				M.emote("cough")
-			addtimer(VARSET_CALLBACK(M, coughedtime, 0), 2 SECONDS)
+			affected_mob.drop_held_item()
+		affected_mob.apply_damage(1, OXY)
+		if(affected_mob.coughedtime < world.time && !affected_mob.stat)
+			affected_mob.coughedtime = world.time + 2 SECONDS
+			if(ishuman(affected_mob)) //Humans only to avoid issues
+				affected_mob.emote("cough")
+
+/////////////////////////////////////////////
+// Miasma smoke (for LZs)
+/////////////////////////////////////////////
+
+/obj/effect/particle_effect/smoke/miasma
+	name = "CN20-X miasma"
+	amount = 1
+	time_to_live = INFINITY
+	smokeranking = SMOKE_RANK_MAX
+	opacity = FALSE
+	alpha = 75
+	color = "#301934"
+	var/burn_damage = 4
+	var/xeno_yautja_multiplier = 3
+
+/obj/effect/particle_effect/smoke/miasma/Initialize(mapload, oldamount, new_cause_data)
+	. = ..()
+	// Mimic dispersal without actually doing spread logic
+	alpha = 0
+	addtimer(VARSET_CALLBACK(src, alpha, initial(alpha)), rand(1, 6) SECONDS)
+
+/obj/effect/particle_effect/smoke/miasma/process()
+	. = ..()
+	for(var/obj/structure/closet/container in get_turf(src))
+		for(var/mob/living/carbon/mob in container)
+			affect(mob)
+
+/obj/effect/particle_effect/smoke/miasma/affect(mob/living/carbon/affected_mob)
+	..()
+
+	var/damage = burn_damage
+	if(isxeno(affected_mob))
+		damage *= xeno_yautja_multiplier
+	else if(isyautja(affected_mob))
+		if(prob(75))
+			return FALSE
+		damage *= xeno_yautja_multiplier
+
+	affected_mob.apply_damage(damage, BURN)
+	affected_mob.AdjustEyeBlur(0.75)
+
+	if(affected_mob.coughedtime < world.time && !affected_mob.stat)
+		affected_mob.coughedtime = world.time + 2 SECONDS
+		if(ishuman(affected_mob)) //Humans only to avoid issues
+			if(prob(50))
+				affected_mob.emote("cough")
+			else
+				affected_mob.emote("gasp")
+			if(prob(20))
+				affected_mob.drop_held_item()
+		to_chat(affected_mob, SPAN_DANGER("Something is not right here..."))
```

Comment:
is it lore accurate that synths can cough?

Reply:
¯\\_(ツ)_/¯ Nothing stopping them from faking or programming to just do it. But regardless synths will not be immune to this smoke for gameplay reasons. If someone cares, they can fix all the other situations that don't synth check in this file & there is an emote.

- /obj/effect/particle_effect/smoke/xeno_weak_fire
- /obj/effect/particle_effect/smoke/xeno_burn
- /obj/effect/particle_effect/smoke/phosphorus
- /obj/effect/particle_effect/smoke/sleepy
- /obj/effect/particle_effect/smoke/bad

Reply:
No reason it can't be lore friendly, synthetics are not immune to several environmental factors, so it can be just fluffed up this upgraded chemical weapon also works on Synthetic tissue.

Reply:
> ¯\_(ツ)_/¯ Nothing stopping them from faking or programming to just do it. But regardless synths will not be immune to this smoke for gameplay reasons. If someone cares, they can fix all the other situations that don't synth check in this file & there is an emote.
> 
> * /obj/effect/particle_effect/smoke/xeno_weak_fire
> * /obj/effect/particle_effect/smoke/xeno_burn
> * /obj/effect/particle_effect/smoke/phosphorus
> * /obj/effect/particle_effect/smoke/sleepy
> * /obj/effect/particle_effect/smoke/bad

I ain' gonna do it